__tests__/app.test.js

```diff
@@ -167,3 +167,83 @@ describe("GET /api/articles/:article_id/comments", () => {
       });
   });
 });
+
+describe("POST /api/articles/:article_id/comments", () => {
+  test("201: inserts a new comment and returns the inserted comment to the client ", () => {
+    const newComment = { username: "butter_bridge", body: "cool stuff" };
+    return request(app)
+      .post("/api/articles/1/comments")
+      .send(newComment)
+      .expect(201)
+      .then(({ body }) => {
+        expect(body.comment).toEqual({
+          comment_id: 19,
+          article_id: 1,
+          votes: 0,
+          created_at: expect.any(String),
+          author: "butter_bridge",
+          body: "cool stuff",
+        });
+      });
+  });
```

Comment:
What happens if the comment is submitted with an addtional property but username and body are fine?

Reply:
I've added a test for this